src/fontra/client/core/actions.js

```diff
@@ -187,6 +223,9 @@ function loadActionsByKeyOrCode() {
   actionsByKeyOrCode = {};
   for (const [actionIdentifier, action] of Object.entries(actionInfoController.model)) {
     for (const shortCut of action.customShortCuts || action.defaultShortCuts || []) {
+      if (!shortCut) {
```

Comment:
How can this happen?

Reply:
That was related to customShortCuts being `null`. I changed it to be an empty list instead. The check `if (!shortCut)` is not needed anymore.